crowbar_framework/app/models/service_object.rb

```diff
@@ -1408,6 +1423,17 @@ def apply_role(role, inst, in_queue)
     # XXX: This should not be done this way.  Something else should request this.
     system("sudo", "-i", Rails.root.join("..", "bin", "single_chef_client.sh").expand_path.to_s) if !ran_admin
 
+    # are there any roles to remove from the runlist?
+    role_to_remove = databag["deployment"][@bc_name]["elements"].select {|r| r =~ /_remove/}.keys.first
```

Comment:
Metrics/LineLength: Line is too long. [103/80]<br>Style/SpaceInsideBlockBraces: Space between { and | missing.<br>Style/SpaceInsideBlockBraces: Space missing inside }.